chains/polygon-pos/server/src/eth_types.rs

```diff
@@ -0,0 +1,93 @@
+use ethers::types::{Bytes, H160, U256, U64};
+use serde::{Deserialize, Serialize};
+
+pub const FEE_OP_TYPE: &str = "FEE";
+pub const CALL_OP_TYPE: &str = "CALL";
+pub const MINING_REWARD_OP_TYPE: &str = "MINER_REWARD";
```

Comment:
do we need to copy everything? I think just creating a new config crate should be enough